code/controllers/master.dm

```diff
@@ -297,7 +297,7 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 		SS.queue_next = null
 		SS.queue_prev = null
 		SS.state = SS_IDLE
-		if (SS.flags & SS_TICKER)
+		if ((SS.flags & (SS_TICKER|SS_BACKGROUND)) == SS_TICKER)
```

Comment:
It seems strange that this would be needed, you sure it is?

Reply:
not really, but i do know that when ssexplosion tells the mc to halt processing subsystems until the next tick, it assumes subsystems in this list still needs to be ran and checked, and i figured background subsystems aren't likely to be intended to count for that since the goal is to reduce lag and allocate more time to byond when we know that a bunch of turf updates are about to go out.